contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -249,6 +249,17 @@ rewrite_object_refs(Node *stmt)
 								 errmsg("Dropping members to db_owner is not currently supported "
 										"in Babelfish")));
 				}
+				else if (strcmp(role_name, DB_ACCESSADMIN) == 0)
+				{
+					if (has_privs_of_role(GetUserId(), get_role_oid(get_db_owner_name(db_name), false)))
+					{
+						/* only members of db_owner can alter drop members to fixed db roles */
+					}
+					else
```

Comment:
Why not write a standalone inverse `if` instead of this empty `if` and then else? Like:
```
if (!has_privs_of_role(GetUserId(), get_role_oid(get_db_owner_name(db_name), false)))
```
Anyways, why don't we do it it in processUtility instead of object_rewrite?

Reply:
> Why not write a standalone inverse if instead of this empty if and then else? Like:

Readability, these are important permission checks and reading inverse condition felt a bit confusing.

> Anyways, why don't we do it it in processUtility instead of object_rewrite?

Doing it in object_rewrite meant we just need to compare the original name. Doing it in processUtility means we will need to fetch the original name from the physical name using babelfish catalog.
But since we have syscache now, I'll do that.